frontend/src/service/eventService.js

```diff
@@ -6,7 +6,26 @@ const API_getAllEvents = () => {
   return axios
     .get('/api/event', createHeader(localStorage.getItem('JWT')))
     .then(response => response.data)
-    .catch(err => showError(err.message, 'Cant fetch Notification User Data.'))
+    .catch(err => showError(err.message, 'Can´t fetch Notification User Data.'))
```

Comment:
```suggestion
    .catch(err => showError(err.message, `Can't fetch Notification User Data.`))
```
pretty much everywhere where you use can't or any other apostroph